### ISSUE TITLE
Docs don't make location of GhApi clear

### ISSUE THREAD

**Issue by henryiii** [fastai/ghapi#26]
I could not figure out where `GhApi` came from just by reading the docs, I tried `ghapi.GhApi`, I tried `dir(ghapi)`, and no sign of it. I clicked through a bit and didn't see it either. Finally found it by entering `colab` and seeing the `ghapi.core` import at the top that gets hidden. Two recommendations:

* Somewhere in the docs mention that `ghapi.core` is the home for GhApi.
* Use qualified references in the docs. `import *` makes docs really hard to follow (especially when it's then hidden! But even not hidden it is bad in docs). Since most operations are on the `ghapi.core.GhApi` object, it won't make the docs that much more verbose, and would be much easier to follow.

PS: It's also missing from the readme.

**Comment by hamelsmu** [comment]
Added this to the README/docs

```py
from ghapi.all import GhApi
```

cc: @jph00
